src/pkg/egress/syslog/syslog_connector_test.go

```diff
@@ -219,6 +219,34 @@ var _ = Describe("SyslogConnector", func() {
 			Eventually(logClient.sourceInstance).Should(HaveKey("3"))
 		})
 
+		It("doesn't emit LGR and SYS log to the log client about aggregate drains drops", func() {
```

Comment:
Not sure why we emit them under both source types to be honest. I'd think maybe one only would/should be fine to be honest, but didn't feel like changing that. Why not for aggregate drains? While a app developer might be tailing/looking for logs using the cli, I don't believe operators are going to be looking for drops using cf tail/cf logs. Especially because those drops arn't going to be emitted under a application id, so only cf tail would work.